cegui/include/CEGUI/RendererModules/Ogre/Renderer.h

```diff
@@ -496,6 +499,41 @@ class OGRE_GUIRENDERER_API OgreRenderer : public Renderer
     */
     bool usesDirect3D();
 
+#ifdef CEGUI_USE_OGRE_COMPOSITOR2
+    enum RenderingModes
```

Comment:
Cegui use C++11, so maybe scoped by `enum class` instead of enumname_enummember?

Reply:
corrected